zoomx/src/main/java/com/zoomx/zoomx/view/request/RequestActivity.java

```diff
@@ -46,7 +50,8 @@ public void setRequestData() {
         viewModel.getRequests().observe(this, new Observer<List<RequestEntity>>() {
             @Override
             public void onChanged(@Nullable List<RequestEntity> requestEntities) {
-                requestAdapter.setRequestEntityList(requestEntities);
+                requestAdapter.setRequestEntityList(requestEntities,
+                        RequestActivity.this, getApplicationContext());
```

Comment:
why have you sent  context object to the adapter ?

Reply:
Removed. not used!